src/components/ui-shell/_platform-side-nav.scss

```diff
@@ -77,41 +78,90 @@ $shell-side-nav-accent-01: $ibm-colors__blue--60;
   }
 
   //----------------------------------------------------------------------------
-  // Header
+  // Title bar
   //----------------------------------------------------------------------------
-  .#{$prefix}--side-nav__header {
+  .#{$prefix}--side-nav__title-bar {
     display: flex;
-    align-items: center;
+    align-items: stretch;
+  }
+
+  .#{$prefix}--side-nav:hover .#{$prefix}--side-nav__title-bar,
+  .#{$prefix}--side-nav--expanded .#{$prefix}--side-nav__title-bar {
+    border-bottom: 1px solid $shell-side-nav-bg-02;
+  }
+
+  // Only show a bottom border for the Icon in the Title bar if we do not have a
+  // sub-menu
+  .#{$prefix}--side-nav__title-bar:not(.#{$prefix}--side-nav__title-bar--sub-menu) > .#{$prefix}--side-nav__icon {
+    border-bottom: 2px solid $shell-side-nav-bg-02;
```

Comment:
change to `border-botton: 1px` (my sketch file was doing something weird so that why it looked like that, there shouldn't be the visible notch under the icon). 
<img width="276" alt="image" src="https://user-images.githubusercontent.com/11670886/45825671-b16ed500-bcea-11e8-86d2-e08bb7381296.png">